src/internal/_isThenable.js

```diff
@@ -1,6 +1,9 @@
+var _hasMethod = require('./_hasMethod');
+
+
 /**
  * Tests if a value is a thenable (promise).
  */
 module.exports = function _isThenable(value) {
-  return (value != null) && (value === Object(value)) && typeof value.then === 'function';
```

Comment:
Does anyone remember why we included `value === Object(value)`? I recall someone saying this code was copied from another project.